src/common/utils/multiYearPlanHelperFunctions.ts

```diff
@@ -0,0 +1,41 @@
+import { NUM_SEMESTERS, TERM } from 'common/constants';
+import range from 'lodash.range';
+
+export interface SemesterInfo {
+  term: TERM;
+  academicYear: number;
+  calendarYear: number;
+  key: string;
+}
+
+/**
+ * Gets the list of semesters for the multi-year plan table starting with the
+ * given year and specified number of semesters.
+ * This function was separated from getSemestersFromYear to allow easier unit
+ * testing.
+ */
+export const calculateSemesters = (
+  startingAcademicYear: number,
+  numSemesters: number
+):
+SemesterInfo[] => range(numSemesters)
```

Comment:
See note above about using `Array.from()` for this

Reply:
This was also replaced in e2dea8e.